_data/people.yml

```diff
@@ -142,6 +142,20 @@
       icon: linkedin
       name: Linkedin
 
+- name: Oleksandr Yardas
+  pic: /img/people/yardaso.jpeg
```

Comment:
Your jpeg seems to be not square. If it's not square, it'll be an oval. 
![image](https://user-images.githubusercontent.com/393899/90409586-f8001200-e06e-11ea-9914-b365d8c73bb0.png)

Two useful bash commands can help you resize your image. You can use `identify` to establish the current size of your image and the `convert` command from imagemagick to resize the image.

We can see that it's 500x644 pixels. 
``` 
> identify yardaso.jpeg
yardaso.jpeg JPEG 500x644 500x644+0+0 8-bit sRGB 109816B 0.000u 0:00.001
```

By inspecting the image, it's probably best to crop this from the bottom to 500x500. The following command will make that conversion and save it as a new file called yardaso_new.jpeg.

```
> convert yardaso.jpeg -crop 500x500+0+0 yardaso_new.jpeg
```

If you like the result, you can move the new file to overwrite the old file (`mv yardaso_new.jpeg yardaso.jpeg`), commit the new file (`git commit -am "commits cropped square image file for olek"`), and push it to your PR branch (`git push origin source`).

Reply:
Thanks for the mini guide! Learning more about the terminal all the time. The image has been fixed.